src/lib/rpc.ml

```diff
@@ -226,9 +226,22 @@ let dateTime_of_rpc = function
   | DateTime s -> s
   | x -> failwith (Printf.sprintf "Expected DateTime, got '%s'" (to_string x))
 
+let base64_2045_decode s =
+  let open Base64_rfc2045 in
+  let buf = Buffer.create 1024 in
+  let d = decoder (`String s) in
+  let rec go () =
+    match decode d with
+    | `Flush s -> (Buffer.add_string buf s; go ())
+    | `End -> Buffer.contents buf
+    | `Malformed _   (* ignore; usually missing '\r' before '\n' *)
```

Comment:
```suggestion
    (* best-effort *)
    | `Malformed _   (* ignore; usually missing '\r' before '\n' *)
```

Reply:
OK. I agree with you that some kind of parameterization is necesary to handle all the different cases (no line breaks, line breaks every 76 characters with `\r\n`, line breaks every 76 characters with `\n`, etc.). I'll draft a solution and include the suggested comment and documentation. Thanks @mseri.